src/pages/ExcelOpplasting.less

```diff
@@ -117,3 +123,21 @@
     }
   }
 }
+
+@media (min-width: 769px) {
```

Comment:
Er det noen grunn til at det ikke er brukt less variabler her? Ser det er gjennomgående for hele fila...

Reply:
Det må rettes til variabler som Morten nevnte et annet sted, ja :)